utils/touchstone-compare/run_compare.sh

```diff
@@ -22,9 +22,9 @@ set -x
 if [[ ${COMPARE_WITH_GOLD} == "true" ]] || [[ ${COMPARE} == "true" ]]; then
```

Comment:
You removed all documentation for the COMPARE or COMPARE_WITH_GOLD params but we still trigger on them.

Reply:
Indeed, I removed it from the router-perf-v2 workload, that workload does not use those parameters anymore. Network-performance still uses the "old" method.

Reply:
After a second read, I realized you were right, I didn't remove COMPARE_WITH_GOLD variables from the router-perf workload, removing them now.

Reply:
@rsevilla87 do we need this part in utils , since currently network perf does not use utils/touchstone-compare and when we will refactor network-perf we will use the standardized version, so is this required?